alphadia/data/alpharaw_wrapper.py

```diff
@@ -4,23 +4,13 @@
 import os
 
 import numba as nb
-
-# third party imports
 import numpy as np
 import pandas as pd
-from alpharaw import mzml as alpharawmzml
-
-# TODO fix: "import resolves to its containing file"
-from alpharaw import sciex as alpharawsciex
-
-# alpha family imports
-from alpharaw import (
-    thermo as alpharawthermo,
-)
+from alpharaw import mzml as alpharaw_mzml
```

Comment:
kind of unrelated, but this totally confused my IDE

Reply:
mission accomplished :D